tests/integ/scala/test_window_frame_suite.py

```diff
@@ -251,20 +265,30 @@ def test_range_between_should_accept_at_most_one_order_by_expression_when_unboun
         df.select(
             min_("key").over(window.range_between(Window.unboundedPreceding, 1))
         ).collect()
-    assert "Cumulative window frame unsupported for function MIN" in str(ex_info)
+        if not local_testing_mode:
+            assert "Cumulative window frame unsupported for function MIN" in str(
```

Comment:
The error message doesn't make sense. This error is caused by `window.range_between(Window.unboundedPreceding, 1)`, you cannot specify range between for non-cumulative window frames. As an evidence, changing the expression to `window.range_between(Window.unboundedPreceding, 0)` runs regardless of the number of order expressions.